dlc-manager/tests/test_utils.rs

```diff
@@ -714,3 +715,59 @@ pub fn refresh_wallet<B: Deref, W: Deref>(
         retry += 1;
     }
 }
+
+/// Serialize an hexadecimal value.
+pub fn serialize_hex<S>(hex: &[u8], s: S) -> Result<S::Ok, S::Error>
```

Comment:
I think there are the same methods in `dlc::serde_utils`, maybe easier to just reference that?

Reply:
Yep, will do that!